src/Kconfig.coap

```diff
@@ -40,12 +40,42 @@ config GOLIOTH_MAX_NUM_OBSERVATIONS
     help
         The maximum number of CoAP paths which can be simultaneously observed.
 
-config GOLIOTH_BLOCKWISE_UPLOAD_BLOCK_SIZE
-    int "Golioth blockwise uploads block size"
-    default 1024
+choice
```

Comment:
Let's add a name like `choice UP_BLOCKSIZE`, so it is possible to override that value with new Kconfig default (e.g. application specific).